lib/theme_check/checks/valid_html_translation.rb

```diff
@@ -11,6 +11,7 @@ class ValidHTMLTranslation < JsonCheck
     def on_file(file)
       return unless file.name.start_with?("locales/")
       return unless file.content.is_a?(Hash)
+      return if file.name =~ /\.schema\./
```

Comment:
Looking at Dawn I'm confused why we opt-out .schema.js of our checks?

Reply:
See https://github.com/Shopify/theme-check/issues/514

Upon brief inspection, those are really not the same.

Reply:
Hmm I think just because it looked like we should (https://github.com/Shopify/theme-check/issues/514). But gotcha, not really a problem for this check particularly.